src/components/autocomplete/autocomplete.stories.js

```diff
@@ -0,0 +1,140 @@
+import { html } from "lit-html";
+import "../autocomplete";
+import { docsIntro } from "../../../.storybook/utils.js";
+
+export default {
+  title: 'custom-elements/vl-autocomplete',
+  parameters: {
+    controls: { hideNoControlsWarning: true },
+    docs: {
+      description: {
+        component: docsIntro({
+          root: 'autocomplete',
+          intro: 'Use autocomplete to show ...',
```

Comment:
Dit kan wel een betere intro gebruiken ;)

Reply:
intro aangepast in branch